sql/src/main/java/org/apache/druid/sql/calcite/rel/DruidQuery.java

```diff
@@ -928,6 +928,7 @@ public GroupByQuery toGroupByQuery()
         sorting != null
         ? new DefaultLimitSpec(
             sorting.getOrderBys(),
+            0,
```

Comment:
Is the plan to update `Sorting` to store the offset, and `computeSorting` to set it when creating it `Sorting` from the calcite `Sort` in a follow-up so that this works in SQL?

Reply:
Yes.